src/python/zquantum/qcbm/cost_function.py

```diff
@@ -0,0 +1,114 @@
+from zquantum.core.interfaces.cost_function import CostFunction
+from zquantum.core.interfaces.backend import QuantumBackend
+from zquantum.core.bitstring_distribution import BitstringDistribution, evaluate_distribution_distance
+from zquantum.core.circuit import build_ansatz_circuit
+from typing import Union, Dict
+import numpy as np
+
+class QCBMCostFunction(CostFunction):
+    """
+    Cost function used for evaluating QCBM.
+
+    Args:
+        ansatz (dict): dictionary representing the ansatz
+        backend (zquantu.core.interfaces.backend.QuantumBackend): backend used for QCBM evaluation
```

Comment:
Missing m in zquantuM 
(sorry I have a thing with typos)

Reply:
Eagle eye, love it!